src/components/ChatEntry.js

```diff
@@ -1,22 +1,69 @@
-import React from 'react';
+import React, {useState} from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+
 
 const ChatEntry = (props) => {
+  const [likeButton, setLikeButton] = useState('🤍');
+
+  const onLikedButtonClick = () => {
+    const updatedMessage = {
+      id: props.id,
+      sender: props.sender,
+      body: props.body,
+      timeStamp: props.timeStamp,
+      liked: !props.liked,
+  
+  // let senderClass = 'chat-entry local'
+  // if (sender === 'Estragon') {
+  //   senderClass = 'chat-entry remote'
+}
+      props.updateMessageData(updatedMessage);
+      setLikeButton(updatedMessage.liked ? '❤️' : '🤍');
+      const bool = likeButton === '🤍';
+      props.updateTotalLikes(bool);
+  };
+
+  const localRemote =
+    props.sender === 'Vladimir' ? 'chat-entry local' : 'chat-entry remote';
+
+
+
+
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={localRemote}>
```

Comment:
The only part of the class that changes based on being the local or remote sender is `local` or `remote`. `chat-entry` doesn't change. So consider only storing `local` or `remote` in `localRemote`, then modify this to
```js
    <div className={`chat-entry ${localRemote}`}>
```

This can help us avoid needing to build a huge combination of class name strings if we had other classes to toggle between.